### PR TITLE
salar | Refactor/gd to functional

### PR DESCRIPTION
**Pull request by salarhfz-fs**
This PR is to refactor Google Drive helper components to functional.
Including:
- **src/common/integrations/GoogleDrive.js**
- **src/botPage/view/react-components/Integrations/GoogleDriveIntegration.js**
- **src/botPage/view/deriv/layout/ToolBox/components/load/load.jsx**
- **src/botPage/view/deriv/layout/ToolBox/components/save/save.jsx**

Also fixes some of the _TrackJS_ errors we already face.
It will enable **Google Drive** testing in _local_, _staging_ and _test-link deployments_ too.
And puts the environment secrets to `.env` to prevent access by 3rd-parties.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,22 +47,36 @@ For Windows:
...
-### 3. Starting a Development Server
+### 3. Preparing environment variables
+If you want to use *Google Drive features* in the project (**to Save/Load strategies**)
+you need to create a file called `.env` at the project root with the following content:
+```
+GD_CLIENT_ID=GOOGLE_DRIVE_CLIENT_ID
+GD_API_KEY=GOOGLE_DRIVE_API_KEY
+GD_APP_ID=GOOGLE_DRIVE_APP_ID
+```
+Here comes useful links for more information on how to confgure your project
```
Comment (sara-fs) [inline README.md:58]: typo in `configure`

Comment (salarhfz-fs) [reply README.md]: Fixed :+1:

**Review by mehrzad-fs — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -51,6 +51,13 @@ commands:
   build:
     description: "Build"
     steps:
+      - run:
+          name: "Create environment variables file"
+          command: |
+            echo GD_CLIENT_ID=${GD_CLIENT_ID} > .env
+            echo GD_API_KEY=${GD_API_KEY} >> .env
```
Comment (sandeep-deriv) [inline .circleci/config.yml:58]: What is `>` and `>>`
And what is difference?

Comment (salarhfz-fs) [reply .circleci/config.yml:58]: These are `redirection` utilities in shell that allows us to redirect standard output (`stdout`) to another stream except than terminal itself.
The first one (`>`) truncates file content and the second (`>>`) appends to a file while writing content.

**Review by sandeep-deriv — DISMISSED** [review]

**Review by sara-fs — APPROVED** [review]

**Review by mehrzad-fs — APPROVED** [review]

**Pull request merged**